feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/data/network/subscan/WalletNetworkApi.kt

```diff
@@ -22,10 +26,8 @@ interface SubscanNetworkApi {
         @Body body: AssetPriceRequest
     ): SubscanResponse<AssetPriceStatistics>
 
-    @Headers("x-api-key: ${BuildConfig.SUBSCAN_API_KEY}")
-    @POST("//{subDomain}.api.subscan.io/api/scan/transfers")
-    suspend fun getTransactionHistory(
-        @Path("subDomain") subDomain: String,
-        @Body body: TransactionHistoryRequest
-    ): SubscanResponse<TransactionHistory>
+    @POST("http://10.0.2.2:3000/")
```

Comment:
Please add TODO or smth to change it before merging to dev

Reply:
Done